on_call_kit.md

```diff
@@ -10,9 +10,10 @@ It's helpful to have a checklist that you run through prior to going on-call to
 4. Make sure you have access to all the necessary VPNs, password trackers, and related systems. If you don't have a work-issued laptop and you don't want work-related things on your personal machine, set up a machine you can remote into. Ensure that it doesn't go to sleep if it is a Windows machine subject to corporate policy.
 5. Install [f.lux](https://justgetflux.com/). It's worth it for those 3 am pages.
 6. When you're home for the evening, if at all possible, go ahead and have your laptop set up and ready to go. That way you're not fumbling to get it out of your bag and plugged in, etc. at 3 A.M.
-7. A hands-free kit for your phone, whether this be the headphones that came with it or a bluetooth earpiece. Make sure you've tested it with your phone in advance of going on-call. It's good to keep your hands available in case you need to call up a colleague or join a conference call.
-7. A smart watch (pebble, Apple watch, etc.) can be helpful for late night alerts. The vibration of a smart watch will wake only you up where an audible alert from your phone might also wake up your partner as well. Have your charger handy just in case.
-8. A small zippable bag to keep all of your on-call kit in (other than your laptop). Your items, such as chargers, should never leave this bag unless you are using them on call. That way you never have to scramble to be ready for on-call. 
+7. Consider making bookmarks for everything. Save that group of browser tabs with the recipes for restarting things and monitoring tools ordered the way you like it. If there's a set of boxes you like to connect to, save their ssh connection info in your terminal and have everything auto-connect on launch -- 3 A.M. when everything's beeping at you isn't the time to test your memory.
```

Comment:
Ah, should have read the notes more thoroughly! You can just put in a parenthesis like (here's one for Mac OS and Ubuntu) and let others fill in other OSes.